go/src/koding/klientctl/main.go

```diff
@@ -59,6 +59,8 @@ func main() {
 	// always set to a number of available cores.
 	runtime.GOMAXPROCS(runtime.NumCPU())
 
+	debug = debug || config.Konfig.IsDebug()
+
 	// The writer used for the logging output. Either a file, or /dev/null
 	var logWriter io.Writer
 
```

Comment:
@rjeczalik do you know why we handle file writing manually instead of piping them on klient command with `> file 2>&1` ?